Apps/WebClient/src/ClientApp/src/components/timeline/linearTimeline.vue

```diff
@@ -220,23 +222,32 @@ export default class LinearTimelineComponent extends Vue {
         if (this.visibleTimelineEntries.length === 0) {
             return [];
         }
+        const timezoneOffSetInMinutes = new Date().getTimezoneOffset();
         let groups = this.visibleTimelineEntries.reduce<
             Record<string, TimelineEntry[]>
         >((groups, entry) => {
             // Get the string version of the date and get the date
-            const date = new Date(entry.date).setHours(0, 0, 0, 0);
-
+            const localDate = new Date(entry.date);
+            debugger;
+            const dateGroup = localDate.setHours(
```

Comment:
example: entry.date = "2020-08-12T00:00:00"
const localDate = new Date(entry.date); and localDate.setHours basing on the timezone off set will make sure the dateGroup is the right date (2020-08-12) instead of 2020-08-11.